ibm_catalog.json

```diff
@@ -1,200 +1,214 @@
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-container-registry/main/README.md",
+      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-container-registry/main/images/icr_icon.svg",
+      "provider_name": "IBM",
+      "features": [
         {
-            "name": "deploy-arch-ibm-container-registry",
-            "label": "Cloud automation for IBM Container Registry",
-            "product_kind": "solution",
-            "tags": [
-                "ibm_created",
-                "target_terraform",
-                "terraform",
-                "containers",
-                "registry",
-                "solution"
-            ],
-            "keywords": [
-                "containers",
-                "IaC",
-                "infrastructure as code",
-                "terraform",
-                "solution",
-                "registry"
-            ],
-            "short_description": "Creates or uses an existing IBM Container Registry namespace, configures pull traffic and storage quotas, and supports upgrading the registry plan to Standard.",
-            "long_description": "This architecture creates or utilizes an existing IBM Container Registry namespace, provides the ability to configure pull traffic limits and storage quotas in megabytes, and allows for upgrading the registry plan to Standard. It ensures efficient management of container image access by regulating data pull volume from the registry and setting storage capacity limits for container images within each registry.",
-            "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-container-registry/main/README.md",
-            "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-container-registry/main/images/icr_icon.svg",
-            "provider_name": "IBM",
+          "title": "Creates or Uses Existing IBM Container Registry Namespace",
+          "description": "Enables the creation of a new IBM Container Registry namespace or the use of an existing one, allowing users to define isolated environments for managing container images, with the ability to set a retention policy."
+        },
+        {
+          "title": "Configures Pull Traffic Limits",
+          "description": "Allows fine-grained configuration of pull traffic limits in megabytes, controlling the amount of data that can be pulled from the registry."
+        },
+        {
+          "title": "Sets Storage Quotas",
+          "description": "Enables the configuration of storage quotas in megabytes, defining the maximum amount of storage available for container images within each registry."
+        },
+        {
+          "title": "Upgrades Registry Plan to Standard",
+          "description": "Provides the ability to upgrade the container registry plan to Standard, allowing for enhanced features and capabilities."
+        }
+      ],
```

Comment:
Missing support details. EG: https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager/blob/058c06d535e2cab832018051d18c367f04f5a93d/ibm_catalog.json#L53

Reply:
can you add the terraform version too?
https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/blob/bcda6af62a53987dcdc52545f289dda29e56950c/ibm_catalog.json#L305